regparser/commands/compare_to.py

```diff
@@ -44,17 +44,19 @@ def compare(local_path, remote_url):
 
         if remote != local:
             click.echo("Content differs: {} {}".format(local_path, remote_url))
-            if click.confirm("Show diff?"):
+            if not prompt or click.confirm("Show diff?"):
                 diffs_str = '\n'.join(udiff(remote, local))
-                click.echo_via_pager(diffs_str)
+                (click.echo_via_pager if prompt else click.echo)(diffs_str)
```

Comment:
In the interest of time and not refactoring unit tests, I made this easier to read without changing the responsibilities of the function. Happy to move the prompt and echo to `compare_to` if you prefer though.